src/main/java/examples/Examples.java

```diff
@@ -410,4 +410,24 @@ public void example12(MongoService mongoService) {
 
   }
 
+  public void example13_0(MongoService mongoService) {
+
+    JsonObject document = new JsonObject().put("title", "The Hobbit")
+        .put("publicationDate", new JsonObject().put("$date", -1018659600000L /* 21/9/1937 */));
```

Comment:
Also according to the Mongo Java driver docs it says it supports extended JSON too:

http://mongodb.github.io/mongo-java-driver/3.1/bson/extended-json/

So I'm not sure I'd agree that just using a long (as done in this PR) is remaining as close as possible to the Java driver behaviour.